frontend/src/wizards/Placement/MatchExpression.css

```
@@ -0,0 +1,9 @@
+.match-expression-field .pf-v6-c-menu-toggle:not(.pf-m-typeahead) {
+  height: calc(
+    var(--pf-t--global--font--size--body--default) * var(--pf-t--global--font--line-height--body) + 2 *
```

Stylelint (17.7.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)